flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/hint/FlinkHintStrategies.java

```diff
@@ -122,6 +119,20 @@ private static HintOptionChecker fixedSizeListOptionChecker(int size) {
                             FlinkHints.stringifyHints(Collections.singletonList(hint)),
                             hint.hintName);
 
+    private static final HintOptionChecker OPTIONS_KV_OPTION_CHECKER =
```

Comment:
why we need to introduce such refactor?

Reply:
It seems this comment is not addressed.

Reply:
Added more checks for new paramters, It's just a checker like others.

Reply:
Got it, and please takes a look at the returned value.